src/main/java/me/qoomon/gitversioning/GitVersioning.java

```diff
@@ -86,4 +93,21 @@ public static GitVersionDetails determineVersion(
                 gitVersion
         );
     }
+
+    private static String dateTimeOrEmpty(Date headCommitDate){
+        if (headCommitDate == null){
+            return "";
```

Comment:
I would prefer to return a zero version of `VERSION_DATE_TIME_FORMAT` like "00000000.000000" instead of "" because an empty string would look strange in a version pattern like this "{branch}-{commit.date}"
I think we should define another variable `NO_VERSION_DATE_TIME="00000000.000000"`